packages/@sanity/default-login/src/LoginWrapper.js

```diff
@@ -30,13 +30,15 @@ export default class LoginWrapper extends React.PureComponent {
       PropTypes.string,
       PropTypes.node
     ]),
-    sanityLogo: PropTypes.node
+    sanityLogo: PropTypes.node,
+    loadingScreen: PropTypes.node,
   }
 
   static defaultProps = {
     title: 'Choose login provider',
     description: null,
     sanityLogo: <SanityStudioLogo />,
+    loadingScreen: <Spinner center fullscreen />
```

Comment:
I think both of these props (`sanityLogo` and `loadingScreen`) should be components, node actual nodes?

Reply:
Hm maybe `PropTypes.node` is wrong. `PropTypes.element` does not give any error.  Or should it be on only functions?

Reply:
Please note that this is a breaking change, and things around needs to be updated if the sanityLogo proptype is changed.

Reply:
Example where we use this:
https://github.com/sanity-io/manage-sanity-io/blob/26f4ff5c6a09f6604a8214e71448fb902c921b7d/src/routes/invite/Invite.js#L191

Reply:
I don't think it's a great idea to pass "pre-rendered" elements around, but if we want to maintain backwards compatibility we could do something along the lines of:

```js
const LoginWrapper = () => (
  <div>
    {/* ... */}
    {typeof LoadingScreen === 'function' ? <LoadingScreen /> : LoadingScreen}
  </div>
)
```

Then set the prop type to `PropTypes.oneOfType([PropTypes.func, PropTypes.node])`. Thoughts?